ts/screens/authentication/components/PictogramOnSmallDevice.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  IOColors,
+  IOPictograms,
+  IOPictogramSizeScale,
+  IOStyles,
+  Pictogram
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { Dimensions, View } from "react-native";
+
+export const MIN_HEIGHT_TO_SHOW_FULL_RENDER = 820;
+
+type IOPictogramsProps = {
+  name: IOPictograms;
+  color?: IOColors;
+  pictogramStyle?: "default" | "light-content" | "dark-content";
+  size?: IOPictogramSizeScale | "100%";
+};
+
+/**
+ * if the device height is > 800 then the pictogram will be visible,
+ * otherwise it will not be visible
+ * @param pictogramName
+ * @returns Pictogram JSX element
+ */
+
+const PictogramOnSmallDevice = (Props: IOPictogramsProps) => (
```

Comment:
Not entirely convinced by the component name, as it seems to be a component that shows the `Pictogram` on small devices. I don't know which is better, but I think `PictogramHiddenOnSmallDevice` is clearer. What do you think?

Reply:
The name didn't convince me either, so thanks, I will apply this suggestion!